internal/engine/filter/convert.go

```diff
@@ -226,11 +230,16 @@ func joinQueries(f1, f2 string) string {
 	return fmt.Sprintf("%s %s", f1, f2)
 }
 
-func joinStrings(escape bool, a ...string) string {
+func joinStrings(escape bool, a ...string) (string, error) {
 	if escape {
-		return joinEscaped(a...)
+		for _, a := range a {
+			if strings.Contains(a, `"`) {
+				return "", fmt.Errorf("invalid quote in %q.", a)
```

Comment:
golang-ci-lint complains here:

```
error-strings: error strings should not be capitalized or end with punctuation or a newline (revive)
```

Please remove the `.` at the end of the error message.

Reply:
force pushed the fix